src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -2,16 +2,17 @@
 frmelmnts.lbl.noMatchingRecordFound = There are no participants in this batch
 frmelmnts.lbl.offline = You are offline
 frmelmnts.lbl.online = You are online
-frmelmnts.lbl.connectInternet = Please connect to the internet to view content
+frmelmnts.lbl.connectInternet = To browse content, connect to the Internet
 frmelmnts.btn.contentImport = Upload Files
 frmelmnts.btn.selectContentFiles = Select file(s)
-frmelmnts.lbl.uploadEcarFromPd=Upload .ecar files from a pen drive to import content
+frmelmnts.lbl.uploadEcarFromPd=Upload DIKSHA files (eg: MATHS_01.ecar) from your pen drive
```

Comment:
wherever diksha is there replace it with dynamic instance from input hidden from html